src/main/java/org/ice4j/ice/harvest/SinglePortUdpHarvester.java

```diff
@@ -303,21 +302,19 @@ private void runInHarvesterThread()
                     catch (SocketException se)
                     {
                         logger.info("Could not create a socket: " + se);
-                        continue;
```

Comment:
I removed them because IntelliJ IDEA warned about them. I personally agreed with IntelliJ IDEA because the continues made me expect that there was more code bellow but it turned out there wasn't.